src/account_history.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use dialoguer::History;
+use std::collections::VecDeque;
```

Comment:
Can you put std first, then newline, then dialoguer to fit our style please 🙏🏻 ?

Reply:
done